helix-view/src/editor.rs

```diff
@@ -29,6 +29,18 @@ pub struct Config {
     pub scroll_lines: isize,
     /// Mouse support. Defaults to true.
     pub mouse: bool,
+    /// Line number mode.
+    pub line_number: LineNumber,
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+pub enum LineNumber {
+    /// Show absolute line number
+    Absolute,
+
+    /// Show relative line number to the primary cursor
+    Relative,
```

Comment:
Should we add `None` as well?

Reply:
I'd rather configure this with a `gutters = [diagnostics, line_number]` setting